olm.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+operator-sdk generate bundle \
+    --package minio-directpv \
+    --version "$RELEASE" \
+    --deploy-dir resources/base \
```

Comment:
```suggestion
    --deploy-dir resources/base/"$RELEASE" \
```

Can we structure the `resources/base` directory in the following way

```
resources/base
|__4.0.3
|    |__ directpv.yaml
|__4.X.X
     |__ directpv.yaml 
```

So, that we can handle versions properly. Because new components might be added / removed in the further versions.